native/src/components/CustomHeaderButtons.tsx

```diff
@@ -47,7 +47,8 @@ const CustomHeaderButtons = (props: {
       <OverflowMenu
         onPress={onOverflowMenuPress(cancelLabel)}
         accessibilityLabel={t('moreOptions')}
-        OverflowIcon={<Icon Icon={MenuIcon} />}>
+        OverflowIcon={<Icon Icon={MenuIcon} />}
+        style={{ marginRight: 7 }}>
```

Comment:
We try to avoid inline styling whenever possible and instead use [styled components](https://styled-components.com/docs/basics#getting-started)

Reply:
@LeandraH Thank you for explaining the pr thing :) As I understood, I do not need to fill in the pr description if there is already the similar issue, I can only link this pr to that issue. Am I right?

Reply:
Hmm, no, we should fill in the template, it's mostly a helper for the people reviewing to make that as easy and simple as possible, plus if we need to find out in the future why we did something, that can help too. Maybe take a look at other people's PRs to see what I mean.

I guess if your PR is part of a bigger set of PRs, there are some cases where you don't need to fill in much of the description, but even then I would recommend explaining a bit of what is happening in the PR.

In this specific case, there's obviously not much to explain so you could just say that you added padding to the icon which seems to have gotten lost somewhere in all the icon refactoring of the last few weeks.